packages/demo/src/index.js

```diff
@@ -39,12 +41,30 @@ async function main() {
   });
 
   // Uncomment out this if you want to demo the player package
-  // const player = webplayer(root);
+  //const player = webplayer(root);
 
-  function load() {
-    player.load(manifestInput.value);
+  async function load() {
+    await player.load(manifestInput.value);
+    populateQualityPicker();
   }
 
+  function populateQualityPicker(){
+    // Reset/Clear-out the drop down menu.
+    while(qualityPicker.options.length > 1){
```

Comment:
I think a bit of a safer variant would be to loop through the options and remove the options without the value `-1`, theoretically you could end up removing auto as is.

Reply:
I see, in case a value ever pushes -1 beyond index 0. 🤔

Reply:
Yes, exactly. Which, of course, isn't likely to ever happen but by being explicit with your intention you are both making the code a bit safer and easier to read :)